tests/mock/Concrete/PlatformOrderDecorator.php

```diff
@@ -0,0 +1,899 @@
+<?php
+
+namespace Mundipagg\Core\Test\Mock\Concrete;
+
+use Mundipagg\Core\Kernel\Abstractions\AbstractModuleCoreSetup as MPSetup;
+use Mundipagg\Core\Kernel\Abstractions\AbstractPlatformOrderDecorator;
+use Mundipagg\Core\Kernel\Interfaces\PlatformInvoiceInterface;
+use Mundipagg\Core\Kernel\Services\MoneyService;
+use Mundipagg\Core\Kernel\ValueObjects\Id\CustomerId;
+use Mundipagg\Core\Kernel\ValueObjects\Id\OrderId;
+use Mundipagg\Core\Kernel\ValueObjects\OrderState;
+use Mundipagg\Core\Kernel\ValueObjects\OrderStatus;
+use Mundipagg\Core\Payment\Aggregates\Address;
+use Mundipagg\Core\Payment\Aggregates\Customer;
+use Mundipagg\Core\Payment\Aggregates\Item;
+use Mundipagg\Core\Payment\Aggregates\Payments\AbstractCreditCardPayment;
+use Mundipagg\Core\Payment\Aggregates\Payments\AbstractPayment;
+use Mundipagg\Core\Payment\Aggregates\Payments\BoletoPayment;
+use Mundipagg\Core\Payment\Aggregates\Shipping;
+use Mundipagg\Core\Payment\Factories\PaymentFactory;
+use Mundipagg\Core\Payment\Repositories\CustomerRepository as CoreCustomerRepository;
+use Mundipagg\Core\Payment\ValueObjects\CustomerPhones;
+use Mundipagg\Core\Payment\ValueObjects\CustomerType;
+use Mundipagg\Core\Payment\ValueObjects\Phone;
+use Mundipagg\Core\Kernel\Services\LogService;
+
+class PlatformOrderDecorator extends AbstractPlatformOrderDecorator
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class PlatformOrderDecorator has 16 non-getter- and setter-methods. Consider refactoring PlatformOrderDecorator to keep number of methods under 10.](https://app.codacy.com/manual/mundipagg/ecommerce-module-core/pullRequest?prid=4926694)